tests/unit/test_charm.py

```diff
@@ -384,3 +394,24 @@ def test_given_self_signed_certificates_not_yet_stored_when_on_certificate_creat
             WaitingStatus("Root Certificates are not yet set"),
             self.harness.charm.unit.status,
         )
+
+    def test_given_self_signed_certificates_with_additional_extensions(self):
```

Comment:
Please follow the test name format used here: given_x_when_y_then_z
- x: Initial situation
- y: Action
- z: Expected behaviour